### PR TITLE
Added list of active organs and organ information page

### LINKED ISSUE #144: Database: Pagina met alle actieve organen
Een pagina met alle actieve organen van GEWIS.

### PR DESCRIPTION
**Pull request by kokx**
Fixes #144 and #143 

Shows basic information about organs

### CODE REVIEW

File: module/Decision/src/Decision/Controller/OrganController.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    /**
+     * Show an organ.
+     */
+    public function showAction()
+    {
+        return new ViewModel(array(
+            'organ' => $this->getOrganService()->getOrgan($this->params()->fromRoute('organ'))
+        ));
```
Comment (jszanto) [inline module/Decision/src/Decision/Controller/OrganController.php]: This should throw a 404 if an organ with the given id doesn't exist.

File: module/Decision/view/decision/organ/show.phtml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <li><?= $membership['member']->getFullName() ?> <?php if (!empty($membership['functions'])): ?> (<?= implode(', ', $membership['functions']) ?>)<?php endif ?></li>
+        <?php endforeach ?>
+        </ul>
+
+        <h2><?= $this->translate('Old members') ?></h2>
+        <ul>
+        <?php foreach ($oldMembers as $member): ?>
+            <li><?= $member->getFullName() ?></li>
+        <?php endforeach ?>
+        </ul>
```
Comment (jszanto) [inline module/Decision/view/decision/organ/show.phtml]: This list could get very long for some organs. Not sure if that is a problem or not.

Comment (kokx) [inline module/Decision/view/decision/organ/show.phtml]: The longest page here is from the AC. And it is about three pages. So it is not a problem at all.

File: module/Decision/src/Decision/Controller/OrganController.php (hunk 1 above)
Comment (kokx) [inline module/Decision/src/Decision/Controller/OrganController.php]: Fixed.

**Pull request merged**